Capri.Web/Controllers/CourseController.cs

```diff
@@ -0,0 +1,91 @@
+﻿using System;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Authorization;
+using Capri.Services.Courses;
+using Capri.Web.ViewModels.Course;
+
+namespace Capri.Web.Controllers
+{
+    [Route("courses")]
+    public class CourseController : Controller
+    {
+        private readonly ICourseCreator _courseCreator;
+        private readonly ICourseUpdater _courseUpdater;
+        private readonly ICourseGetter _courseGetter;
+        private readonly ICourseDeleter _courseDeleter;
+
+        public CourseController(
+            ICourseCreator courseCreator,
+            ICourseUpdater courseUpdater,
+            ICourseGetter courseGetter,
+            ICourseDeleter courseDeleter)
+        {
+            _courseCreator = courseCreator;
+            _courseUpdater = courseUpdater;
+            _courseGetter = courseGetter;
+            _courseDeleter = courseDeleter;
+        }
+
+        [HttpGet("{id:Guid}")]
```

Comment:
That `:guid` is unnecessery. It will be resolved by ASP.NET by argument type

Reply:
Done